scapy/contrib/isis.py

```diff
@@ -317,6 +327,39 @@ class ISIS_IPv6NeighborAddressSubTlv(ISIS_GenericSubTlv):
                    FieldLenField("len", None, length_of= "address", fmt="B"),
                    IP6Field("address", "::")]
 
+class ISIS_AdministrativeGroupSubTlv(ISIS_GenericSubTlv):
+    name = "Administrative Group SubTLV (Color)"
+    fields_desc = [ByteEnumField("code", 3, _isis_subtlv_names_1),
+                   FieldLenField("len", None, length_of="admingroup", fmt="B"),
+                   IPField("admingroup", "0.0.0.1")]
+
+#maxbw value needs to be in BYTES per second
+class ISIS_MaximumLinkBandwidthSubTlv(ISIS_GenericSubTlv):
+    name = "Maximum Link Bandwidth SubTLV"
+    fields_desc = [ByteEnumField("type", 9, _isis_subtlv_names_1),
+                   FieldLenField("len", None, length_of= "maxbw", fmt="B"),
```

Comment:
Can you drop the spaces after `length_of=` in your modifications?

Reply:
Thank you for reviewing. I've eliminated the spaces.